rust/docker-compose.yaml

```diff
@@ -0,0 +1,20 @@
+services:
+  relayer-service:
+    image: gcr.io/abacus-labs-dev/hyperlane-agent:8117739-20250420-201559
+    command: >
+      /bin/sh -c "./relayer
+      --db /relayer/database
+      --relayChains arbitrum,base,bsc,ethereum,optimism
```

Comment:
would be useful to make the `relayChains` and gcr image tag configurable via env vars